pigar/utils.py

```diff
@@ -81,13 +81,25 @@ def print_table(rows, headers=['PACKAGE', 'CURRENT', 'LATEST']):
 
 
 def parse_reqs(fpath):
-    pkg_v_re = re.compile(r'^(?P<pkg>[^><==]+)[><==]{,2}(?P<version>.*)$')
     """Parse requirements file."""
+    pkg_v_re = re.compile(r'^(?P<pkg>[^><==]+)[><==]{,2}(?P<version>.*)$')
+    referenced_reqs_re = re.compile(r'^-r *(?P<req_f>.*)')
     reqs = dict()
     with open(fpath, 'r') as f:
         for line in f:
             if line.startswith('#'):
```

Comment:
Ignore `--extra-index-url` and others are acceptable for now.

Reply:
@damnever Ok. I will leave this part of the code the same then.